distribution/std-lib/Base/src/Data/Text/Extensions.enso

```diff
@@ -12,7 +12,7 @@ polyglot java import org.enso.base.Text_Utils
 ## Computes the number of characters in the text.
 
    A character is defined as an Extended Grapheme Cluster, see Unicode
-   [Standard Annex #29](https://unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries).
+   [Standard Annex 29](https://unicode.org/reports/tr29/Grapheme_Cluster_Boundaries).
```

Comment:
You can't just remove parts of a URL. This is no longer valid. You need to fix the parser itself to deal with arbitrary URL characters inside a URL.